packages/manager/modules/pci/src/projects/project/storages/containers/containers.controller.js

```diff
@@ -59,10 +59,13 @@ export default class PciStoragesContainersController {
       let solution;
       if (!container.s3StorageType) {
         solution = OBJECT_CONTAINER_OFFERS_TYPES.SWIFT;
-      } else if (container.isHighPerfStorage) {
-        solution = OBJECT_CONTAINER_OFFERS_TYPES.HIGH_PERFORMANCE;
       } else {
-        solution = OBJECT_CONTAINER_OFFERS_TYPES.STORAGE_STANDARD;
+addContainerSolution() {
+    this.containers = this.containers.map((container) => ({
+        ...container,
+        containerSolution: !container.s3StorageType ? OBJECT_CONTAINER_OFFERS_TYPES.SWIFT : OBJECT_CONTAINER_OFFERS_TYPES.S3,
+      }));
+}
```

Comment:
I think you have the wrong code here, I think it should be: 
```suggestion
    solution = OBJECT_CONTAINER_OFFERS_TYPES.SWIFT;
```
Or even better remove the if and simply do:
```suggestion
    solution = !container.s3StorageType ? OBJECT_CONTAINER_OFFERS_TYPES.SWIFT : OBJECT_CONTAINER_OFFERS_TYPES.S3;
```